### DISCUSSION TITLE
Cannot access LL as plain JS object in a SvelteKit application

### DISCUSSION THREAD

**Discussion by akkie** [ivanhofer/typesafe-i18n#443] [Q&A] [answered]
Hi,

I currently try to implement this really great library into my SvelteKit application. The documentation says that I can access the LL import as plain JavaScript object: https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/adapter-svelte#ll

This doesn't work for me. The LL import has only the `subscribe` method from the `Readable` interface. Nothing more.

Using the subscribe syntax $LL works as expected.

I use the latest SvelteKit version.

Do I miss something?

Best regards,
Christian

**Comment by akkie** [comment, marked as answer]
Found the solution: https://github.com/ivanhofer/typesafe-i18n/issues/180

Sorry, for the noise.

**Reply by ivanhofer** [reply]
I'll make this more clear by updating the docs
